src/targets/intents/index.jsx

```diff
@@ -1,40 +1,48 @@
 /* global __DEVELOPMENT__ */
 import 'cozy-ui/transpiled/react/stylesheet.css'
 import 'cozy-ui/dist/cozy-ui.min.css'
+import 'styles/intents.styl'
 
 import React from 'react'
+import { Provider as ReduxProvider } from 'react-redux'
 import { render } from 'react-dom'
 import { HashRouter } from 'react-router-dom'
 import { CozyProvider } from 'cozy-client'
 import { I18n } from 'cozy-ui/react/I18n'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
 
 import { CozyProvider as LegacyCozyProvider } from 'lib/redux-cozy-client'
 
 import IntentHandler from 'containers/IntentHandler'
 import { setupAppContext } from '../../appContext'
 
-import 'styles/intents.styl'
-
 document.addEventListener('DOMContentLoaded', () => {
-  const { client, data, store, lang, context } = setupAppContext()
+  const {
+    cozyClient,
+    legacyClient,
+    data,
+    store,
+    lang,
+    context
+  } = setupAppContext()
```

Comment:
It seems we don't return `legacyClient` in setupAppContext https://github.com/cozy/cozy-home/blob/master/src/appContext.js#L42 . Did you forget to push this file?

Reply:
Argh, I split my commits in two, thanks for the catch 🤦

Reply:
I'm adding a test...